rollup.config.js

```diff
@@ -45,19 +46,19 @@ export default (async () => [
         format: "umd",
         name: "singleSpa",
         sourcemap: true,
-        banner: generateBanner("UMD ES5"),
+        banner: generateBanner("UMD"),
```

Comment:
No separate builds for ES5 vs ES2015, since single-spa@7 will no longer support IE11